src/test/fuzz/strprintf.cpp

```diff
@@ -115,28 +115,50 @@ void test_one_input(const std::vector<uint8_t>& buffer)
         case 5:
             (void)strprintf(format_string, fuzzed_data_provider.ConsumeBool());
             break;
-        case 6:
+        }
+    } catch (const tinyformat::format_error&) {
+    }
+
+    if (format_string.find("%") != std::string::npos && format_string.find("c") != std::string::npos) {
```

Comment:
This not just matches `%<some junk>c`, but also `c<some junk>%`, since both searches start from the begin of the format string. To correct that you'd need to save the found position of the `%` (=return value of the first `.find()`) and pass it to the search for the `c` (=second parameter of the second `.find()`). Given that if I understood your intention correctly ;-)
Also, a tiny nit: could use the overloaded version of `.find()` which takes a `char` as first parameter instead of passing strings with length 1, see http://www.cplusplus.com/reference/string/string/find/ (probably more efficient).

Reply:
See comment below :)